providers/google-cloud-storage/src/main/java/org/jclouds/googlecloudstorage/domain/DomainResourceReferences.java

```diff
@@ -32,7 +32,34 @@ private DomainResourceReferences() {
    }
 
    public enum Location {
-      ASIA, EU, US, ASIA_EAST1, US_CENTRAL1, US_CENTRAL2, US_EAST1, US_EAST2, US_EAST3, US_WEST1;
```

Comment:
The documentation suggests that `US_CENTRAL2` exists:

https://cloud.google.com/storage/docs/bucket-locations

Perhaps we should retain the region and mark it as `@Beta`?

Reply:
I had looked at this same page you suggested and had not seen this location with this remark, sorry.
I think that DomainResourceReferences class should have different enums for each storage class because some locations can not be used in certain storage classes.
But I think at the moment this location can be placed back.

Reply:
But there's another factor, I'm working on another system that needs to match the available resources with their respective price, and in Google's price list that region is not found in any resource, it might not be ideal to maintain that region.
This is the price list table offered by Google: https://cloudpricingcalculator.appspot.com/static/data/pricelist.json

Reply:
I prefer not to break any application using this valid region.  Could you special case your code for this situation or consume the suggested `@Beta` annotation?

Reply:
All right, I've put the annotation @Beta, thank you!